packages/components/addon/components/hds/side-nav/index.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="hds-side-nav" ...attributes>
```

Comment:
I considered it to be similar to the "Pagination" components in which the main component is primarily a wrapper (although the Pagination parent components handle other functionality too.)

Reply:
But in case of the `Pagination`, we have a top-level component that can be used as a single drop-in component handling everything (layout **and** logic). At the moment, is unlikely this will happen for the `SideBar`.
And even if it will be possible, considered the animation + responsiveness, it's very likely that there will be a more general over-arching component (not this one) that will handle the whole.